modules/dev-tools/scripts/lint-monorepo.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# Script to check code styles
+
+set -e
+
+prettier-check "{modules,examples,test}/**/*.js" || echo "Running prettier." && prettier --write "{modules,examples,test}/**/*.js" --loglevel warn
```

Comment:
. Could just detect if modules exist, otherwise use src?

Yes, we should unify the scripts (within reason). I wanted to do an integration with a monorepo first so that the needs of both setups were complete.

I tried just adding both src and modules but eslint would stop with an error if any supplied directory is empty :( Couldn't it just have warned?

Reply:
`if [ -d "$DIRECTORY" ]; then` should fix it.